src/screens/Login.tsx

```diff
@@ -11,10 +14,21 @@ export function Login() {
     event.preventDefault();
 
     try {
-      // Axios
-      // console.log(`Username: ${formData.username}, Password: ${formData.password}`);
+      const loginResponse = await loginUser(formData);
+      if (loginResponse.status === 200) {
+        console.log(
```

Comment:
Since react is a `client-side` library. This `logs` are being showed in the user's browser console. Which is considered a bad practice according to [lighthouse](https://developer.chrome.com/docs/lighthouse/best-practices/errors-in-console/).